project1/styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+    background-color: rgb(255, 220, 249);
+    font-family: 'Cute Font', cursive;
+    font-family: 'Lato', sans-serif;
+    font-family: 'Roboto', sans-serif;
+
+}
+
+* {
+    box-sizing: border-box;
+}
+
+main {
+    /* border: 10px solid #444; */
+
+    width: 1310px;
+    padding: 70px;
+    border: 20px solid rgb(255, 199, 232);
+    background-color: rgb(253, 171, 210);
+    text-align: left;
+    margin: 0 auto;
+}
+
+h1 {
+    color: rgb(255, 255, 255);
+    background-color: rgb(255, 73, 158);
+    text-align: center;
+    font-size: 70px;
+    font-weight: bold;
```

Comment:
By default, all the heading elements, including `h1`, are have a bold `font-weight`, so you don't need line 29.